lib/internal/process/execution.js

```diff
@@ -119,7 +119,10 @@ function evalScript(name, body, breakFirstLine, print, shouldLoadESM = false) {
     });
     if (print) {
       const { log } = require('internal/console/global');
-      log(result);
+
+      process.on('exit', async () => {
+        log(await result);
```

Comment:
My concerns would be addressed if we removed the `await` here.
```suggestion
      process.on('exit', () => {
        log(result);
```